RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -192,11 +71,67 @@
-                                                                           'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppIsland_*_*',
-                                                                           'keep recoPhotonCores_islandPhotonCore_*_*',
-                                                                           'keep recoPhotons_islandPhotons_*_*'
-                                                                           ])
-                    )
+    e.toModify( RecoEgammaAOD.outputCommands, 
+                func=lambda outputCommands: outputCommands.extend(['keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppGED_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerpp_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppIsland_*_*',
+                                                                   'keep recoPhotonCores_islandPhotonCore_*_*',
+                                                                   'keep recoPhotons_islandPhotons_*_*'])
+               )
+# RECO content
+RecoEgammaRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gedPhotonCore_*_*',
+        'keep *_gedPhotons_*_*',
+        #'keep *_gedPhotonsTmp_*_*',        
+        'keep recoPhotons_mustachePhotons_*_*',
+        'keep recoPhotonCores_mustachePhotonCore_*_*',
+        #'drop *_gedPhotonsTmp_valMapPFEgammaCandToPhoton_*',
+        'keep recoTrackExtras_ckfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_ckfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfInOutTracksFromConversions_*_*',
+        'keep recoConversions_uncleanedOnlyAllConversions_*_*',
+        'keep recoTrackExtras_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_uncleanedOnlyCkfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfInOutTracksFromConversions_*_*',
+        'keep recoRecoEcalCandidates_hfRecoEcalCandidate_*_*')                                                                 
+)
+RecoEgammaRECO.outputCommands.extend(RecoEgammaAOD.outputCommands)
+
+# Full Event content 
+RecoEgammaFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gsfElectronCores_*_*', 
+        'keep *_gsfElectrons_*_*', 
+        'keep *_uncleanedOnlyGsfElectronCores_*_*', 
+        'keep *_uncleanedOnlyGsfElectrons_*_*', 
+        'keep *_eidRobustLoose_*_*',
+        'keep *_eidRobustTight_*_*',
+        'keep *_eidRobustHighEnergy_*_*',
+        'keep *_eidLoose_*_*',
+        'keep *_eidTight_*_*',
+        'keep *_egmGedGsfElectronPF*Isolation_*_*',
```

Comment:
Can you get rid of the wild-card *?

Reply:
There is no module that related with this line `egmGedGsfElectronPF*Isolation` in any event contents. So it can be deleted.

Reply:
Ok, great! Please do, then

Reply:
According to the gamma sequences run in reco (see ```egmIsolationTask```) this could be split into
```PYTHON
        'keep *_egmGedGsfElectronPFPileUpIsolation_*_*',
        'keep *_egmGedGsfElectronPFNoPileUpIsolation_*_*',
```